octobot_evaluators/evaluators/abstract_evaluator.py

```diff
@@ -129,6 +132,56 @@ def get_required_candles_count(cls, tentacles_setup_config: tm_configuration.Ten
             -1
         )
 
+    def get_context(self, symbol, time_frame, trigger_cache_timestamp,
```

Comment:
I don't think this is a good idea. This breaks the entire design of the Tentacles.